pcweb/components/docpage/navbar/typesense.py

```diff
@@ -149,6 +150,21 @@ def _clear_search_results(self):
         self.search_results = []
         self.show_results = False
 
+    def _preprocess_query(self, query: str) -> str:
```

Comment:
No we dont like this this should be handled in the typesense indexing not a runtime